docs-standards/10-structure-the-content/01-document-types/quickstart-help-me-get-started.md

```diff
@@ -0,0 +1,182 @@
+- Adds new value beyond the title. It should not repeat the title or be a rephrased version of it.
+
+> ⚙️ **Example:**
+> 
+> - *Try out the core features of Waku in minutes.*
+> - *Quickly add payments to your project with Stripe.*
+
+### Overview
+
+Guidelines:
+
+- Start with a brief discussion of this product or feature and its core purposes. Then describe what the user can accomplish in this quickstart.
+- The overview should be concise. Use an additional [concept](./concept-help-me-to-understand.md) article if you need to provide more information.
+- Link to related topics to support the reader's gathering of information.
+
+### "Before you start" section
+
+This section provides the following information:
+
+- The intended audience for this document.
+- The basic knowledge that you expect the user to have before using this quickstart.
+- The software or hardware requirements for the quickstart.
+
+Guidelines:
+
+- Use "Before you start" H2 heading for this section.
+- Write a single bullet list.
+- Provide [links](../../20-style-the-content/10-links.md) to related content such as installation instructions or articles that provide required knowledge.
+- Setting up or installing prerequisites is not part of a quickstart. If you must explain the procedure, use the [Setup](#setup-optional) section.
+
+> ⚙️ **Example:**
+> 
+> - Learn the basics of [Ethereum](https://ethereum.org/en/developers/docs/intro-to-ethereum/) ↗. 
+> - Learn how to use the command line.
+> - Prepare a machine running Ubuntu Linux with the following requirements:
+>   - 4 GB memory
+>   - 2 TB SSD
+>   - Linux 64-bit
+
+### Setup (optional)
+
+Include this section if the product or prerequisites are installed or configured at the same time by the person running the quickstart.
+
+Describe the simplest steps using a numbered list. If more detail is needed, create a separate [procedure](procedure-help-me-to-do.md) article and link to it.
+
+> ℹ️ **Note:** 
+>
+> For additional tips on writing a procedure, see [Procedure docs standards](procedure-help-me-to-do.md
+).
+
+#### Setup heading:
+
+Guidelines:
+
+- Use imperative verbs to describe the section's purpose: *Install*, *Configure*, *Set up* and so on.
+- Use H2 headings for each setup requirement.
+
+> ⚙️ **Example:**
+> 
+> - *Build an nwaku node*
+> - *Sign up for GitHub Copilot*
+
+### Task section
+
+Use this section to describe what users need to do. 
+
+Guidelines:
+
+- Choose two or three tasks that are essential, quick to complete, and provide immediate value to the user. 
+- If your quickstart involves a complex task, break it down into different logical subtasks with each subtask consisting of one or more related steps. 
+
+#### Task heading
+
+Guidelines:
+
+- Focus on the result, not on the task.
+- Start the title with an action verb in the imperative form. Don't use the -ing form of the verb.
+- Use H2 headings for each task.
+- Use H3 headings for each subtask.
+
```

Comment:
What do we want to do with H4 levels? Please write it down.

(Anything beyond H4 is forbidden)